server/controllers/finance/journal/index.js

```diff
@@ -13,10 +13,71 @@
 
 // module dependencies
 const db = require('../../../lib/db');
+const core = require('./core');
+
+var uuid = require('node-uuid');
```

Comment:
Please use `const` for required libraries.  That way you do not accidentally overwrite your variables later in a function!